os/Makefile.unix

```diff
@@ -222,7 +222,11 @@ endif
 ifeq ($(CONFIG_RAW_BINARY),y)
 BIN_EXT = bin
 endif
+ifeq ($(CONFIG_ARCH_ARMV7A_FAMILY),y)
+KERNEL_START=$(CONFIG_FLASH_START)
```

Comment:
Why we should use two config `CONFIG_FLASH_START` and `CONFIG_FLASH_START_ADDR`?
Is the address value of the two confg different?

If the two address values are the same, I think we can do as below.
```
#define CONFIG_FLASH_START CONFIG_FLASH_START_ADDR
```
or change to use CONFIG_FLASH_START_ADDR in armv7-a